app/src/pages/inside/profilePage/profilePage.jsx

```diff
@@ -35,32 +39,82 @@ const messages = defineMessages({
     id: 'ProfilePage.title',
     defaultMessage: 'User profile',
   },
+  profilePageProjectAssignmentTab: {
+    id: 'ProfilePage.projectAssignmentTab',
+    defaultMessage: 'PROJECT ASSIGNMENT',
+  },
+  profilePageProjectApiKeysTab: {
+    id: 'ProfilePage.apiKeys',
+    defaultMessage: 'API KEYS',
+  },
+  profilePageConfigurationExamplesTab: {
+    id: 'ProfilePage.configurationExamples',
+    defaultMessage: 'CONFIGURATION EXAMPLES',
+  },
+});
+
+const getDefaultLinkConfig = (profileRoute) => ({
```

Comment:
Why it is `default` as it is customized for every particular route every time?
Lets name it as `getProfilePageLink`.